TenantFile/Api/Models/TenantPhone.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace TenantFile.Api.Models
+{
+    public class TenantPhone
```

Comment:
Should we make a join table for every Entity to Phone relationship (e.i. TenantPhone, PropertyPhone, OrganizerPhone etc...) or create a base class like Person or Contact that all Entities with PhoneNumbers inherit? This would be ideally be an interface but EF doesn't like those although there might be a way to make it work. 

Alternatively... we could just only allow Tenants to have the many to many relationship with PhoneNumbers

Reply:
Honestly, I think it's acceptable to have just Tenant-Phone being many-to-many right now. I believe it's a unique requirement for just that case.